scripts/jenkins/qa/cloud-mkphyscloud-qa-test-horizon-ui-integration.yaml

```diff
@@ -0,0 +1,71 @@
+- job:
+    name: cloud-mkphyscloud-qa-test-horizon-ui-integration
+    node: cloud-mkphyscloud-gate-qa
+    description: |
+      Runs tempest smoke tests
+      Useful only for cloud that was built by mkcloud.
+      Mandatory parameter: hw_number
+
+    logrotate:
+      numToKeep: 7
+      daysToKeep: -1
+
+    wrappers:
+      - build-name:
+          name: '#${BUILD_NUMBER} - ${scenario_name} - qa$hw_number - horizon-ui'
+      - timestamps
+    publishers:
+      - mkphyscloud-qa-common-publishers
+
+    concurrent: true
+
+    parameters:
+      - string:
+          name: hw_number
+          default: "2"
+          description: Mandatory, number of the QA cloud server
+
+      - string:
+          name: scenario_name
+          description: Optional; scenario name which typically is an integer with a single letter
+
+      - string:
+          name: scenario_job_name
+          description: Optional; name of the scenario jenkins job that is used to trigger this job
+
+      - string:
+          name: scenario_build_number
+          description: Optional; scenario build number that triggered this job
+
+      - bool:
+          name: integrationtests
```

Comment:
Is this parameter needed? This job only triggers the horizon integration tests and nothing else, so no further confirmation for running the test should be required.